### DISCUSSION TITLE
Most of my movies don't show as "available" when they actually are

### DISCUSSION THREAD

**Discussion by mathgoy** [seerr-team/seerr#299] [Q&A]
Hi all,

Currently moving from plex to jellyfin, I am also moving from overseer to jellyseer.
Both Plex and Jellyfin use the same library folders and their libraries are now up to date.

I synced my jellyfin library with jellyseer and, for some reasons, a lot of the movies that are in my jellyfin don't come up as "available" in jellyseer. Therfore, users can make reaquests for those.
The weird part is that overseer shows them as available.

Any idea what I am doing wrong?
Should you need further information, please let me know.

**Comment by fallenbagel** [comment]
I think I know what's happening. Is it possible to jump onto discord so it's easier to debug and discuss? Also please send the log file too

**Reply by mathgoy** [reply]
Hi @Fallenbagel,

I'll connect to discord later today. Thanks a lot for your help

**Comment by Nutmeg0449** [comment]
any news on this? i'm having the same issue

**Reply by fallenbagel** [reply]
This has been fixed for op i think. Could you jump over to discord. It's an easy fix but hard to explain here

**Reply by Nutmeg0449** [reply]
Thanks for the follow up! Turns out I had the task of scan my library set to 24hrs or never. Changed it and now it works. Thanks for the answer!
